notebooks/nowcast.ipynb

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
do you really want to checkin the notebook? (would be the first notebook)

Reply:
Muss nicht aus meiner Sicht, dann behalte ich das lokal.